eotsmanager/cmd/eotsd/daemon/pop.go

```diff
@@ -0,0 +1,319 @@
+package daemon
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcec/v2/schnorr"
+	"github.com/cometbft/cometbft/crypto/tmhash"
+	sdkflags "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	"github.com/spf13/cobra"
+
+	bbnparams "github.com/babylonlabs-io/babylon/app/params"
+	bbntypes "github.com/babylonlabs-io/babylon/types"
+	btcstktypes "github.com/babylonlabs-io/babylon/x/btcstaking/types"
+	"github.com/babylonlabs-io/finality-provider/codec"
+	"github.com/babylonlabs-io/finality-provider/eotsmanager"
+	"github.com/babylonlabs-io/finality-provider/eotsmanager/config"
+	"github.com/babylonlabs-io/finality-provider/log"
+)
+
+const (
+	flagHomeBaby           = "baby-home"
+	flagKeyNameBaby        = "baby-key-name"
+	flagKeyringBackendBaby = "baby-keyring-backend"
+)
+
+func init() {
+	bbnparams.SetAddressPrefixes()
+}
+
+// PoPExport the data needed to prove ownership of the eots and baby key pairs.
+type PoPExport struct {
+	// Btc public key is the eots *bbntypes.BIP340PubKey marshal hex
+	BtcPublicKey string `json:"btcPublicKey"`
+	// Baby public key is the *secp256k1.PubKey marshal hex
+	BabyPublicKey string `json:"babyPublicKey"`
+
+	// Babylon key pair signs adr36
```

Comment:
Why do we need to mention adr36 here?

Reply:
was to correlate how it was signed, but I removed to specify `Babylon key pair signs EOTS public key as hex`